### PR TITLE
Logistic Regression Files, some overfitting changes

### PR DESCRIPTION
**Pull request by BroccoliHijinx**
Within Logistic Regression, I have a table that I cannot get working.  I want to keep messing around with it, but I'm not sure what is wrong.  I am using the rst basic table, but I think the spacing is off somehow.

### CODE REVIEW

**Comment by BroccoliHijinx** [comment]
@hokie45

**Comment by BroccoliHijinx** [comment]
Here's a link to the tutorial I am using for the table that I can't get to work. http://docutils.sourceforge.net/docs/user/rst/quickref.html#tables

File: docs/source/content/overview/overfitting.rst [hunk 1]
```diff
@@ -9,25 +9,22 @@ Overfitting and Underfitting
...
 
 ----------------------------
 Overfitting
 ----------------------------
 
-A model suffers from **Overfitting** when it has learned too much from the training data, and does not perform well in practice as a result.  This is usually caused by the model having too much exposure to the training data.  For the number classification example, if the model is overfit in this way, it may be picking up on tiny details that are misleading, like stray marks as an indication of a specific number.  Think of overfitting as looking like this:
-
-.. figure:: _img/Lagrange_Error.svg.png
+A model suffers from **Overfitting** when it has learned too much from the training data, and does not perform well in practice as a result.  This is usually caused by the model having too much exposure to the training data.  For the number classification example, if the model is overfit in this way, it may be picking up on tiny details that are misleading, like stray marks as an indication of a specific number.
```
Comment (hokie45) [inline docs/source/content/overview/overfitting.rst]: Could you add a picture for overfitting

File: docs/source/content/overview/overfitting.rst [hunk 2]
```diff
@@ -9,25 +9,22 @@ Overfitting and Underfitting
...
 
 ----------------------------
 Underfitting
 ----------------------------
 
-A model suffers from **Underfitting** when it has not learned enough from the training data, and does not perform well in practice as a result.  As a direct contrast to the previous idea, this issue is caused by not letting the model learn enough from training data.  In the number classification example, if the training set is too small or the model has not had enough attempts to learn from it, then it will not be able to pick out key features of the numbers.  Think of underfitting as looking like this:
+A model suffers from **Underfitting** when it has not learned enough from the training data, and does not perform well in practice as a result.  As a direct contrast to the previous idea, this issue is caused by not letting the model learn enough from training data.  In the number classification example, if the training set is too small or the model has not had enough attempts to learn from it, then it will not be able to pick out key features of the numbers.
```
Comment (hokie45) [inline docs/source/content/overview/overfitting.rst:26]: Same thing, can you add a picture, and explain why its underfitting

File: docs/source/content/overview/overfitting.rst (hunk 1 above)
Comment (hokie45) [inline docs/source/content/overview/overfitting.rst:18]: Could you add a picture of overfitting, and expain why it's overfitting

File: docs/source/content/overview/overfitting.rst [hunk 3]
```diff
@@ -9,25 +9,22 @@ Overfitting and Underfitting
...
 
-.. figure:: _img/Underfit_Graph.PNG
 
 The issue with this estimate is clear to the human eye, the model should be nonlinear, and is instead just a simple line.  In machine learning, this could be a result of underfitting, the model has not had enough exposure to training data to adapt to it, and is currently in a simple state.
```
Comment (hokie45) [inline docs/source/content/overview/overfitting.rst:30]: Can you add a picture of what a good fit looks like?

File: docs/source/content/supervised/logistic_regression.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+When to use it
+-----------------
+
+Logistic Regression is great for situations where you need to classify between two categories.  
+Some good examples are accepted and rejected applicants and victory or defeat in a competition.  
+Here is an example table of data that would be a good candidate for logistic regression.
+
+=====  ======   ======= 
+  Studying      Success
```
Comment (hokie45) [inline docs/source/content/supervised/logistic_regression.rst:29]: Yeah im not sure why this table isn't working, you might just have to make it a picture or something

File: code/supervised/Logistic_Regression/logistic_ex1.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+from sklearn.datasets import load_iris
```
Comment (hokie45) [inline code/supervised/Logistic_Regression/logistic_ex1.py:1]: When i run the code i get this "[0 0]
[[9.81797141e-01 1.82028445e-02 1.44269293e-08]
 [9.71725476e-01 2.82744937e-02 3.01659208e-08]]
0.9733333333333334". 
Then i also get an warning"ConvergenceWarning: lbfgs failed to converge. Increase the number of iterations.
  "of iterations.", ConvergenceWarning)".
Maybe you can just explain what these numbers mean, and if you also get this error maybe explain what it is

File: code/overview/overfitting/example.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
+print(__doc__)
```
Comment (hokie45) [inline code/overview/overfitting/example.py:1]: I would add some more comments to these explaining what some of the lines do

**Review by hokie45 — CHANGES_REQUESTED** [review]

File: code/overview/overfitting/example.py (hunk 6 above)
Comment (mergu) [inline code/overview/overfitting/example.py:1]: this code is copied directly from 
https://scikit-learn.org/stable/auto_examples/model_selection/plot_underfitting_overfitting.html

we should be writing our own code examples for this stuff

**Review by mergu — CHANGES_REQUESTED** [review]

**Review by astorfi — CHANGES_REQUESTED** [review]
Please comment on the codes.

**Review by mergu — CHANGES_REQUESTED** [review]
As stated on other PRs, we should be using a line length limit. As it stands, every paragraph here is just one long line. Typically the limit is 80-100 chars

**Comment by astorfi** [comment]
I will merge it for my records.

**Pull request merged**